Solucion_alumnoAaron

12Aaron opened the pull request:
Posible Solucion (no se si esta bien)

12Aaron committed:
Create Solucion.py

Solución de alumno Aaron de Mil Programadores salta

12Aaron committed:
Solucion_Aaron

Posible solución(No se si esta bienjj)

enzzo19 commented:
Hola Aron buenas, mira la funcion devuelve un espaciado extra despues de la coma cuando ingresa la cadena de la consigna, por lo que "H o l a ,   T ú" no es lo mismo que "H o l a ,   T ú" intenta ponerle a la funcion las instrucciones para que se vea como lo pide la consigna y si necesitas ayuda lo podemos ver 👍

12Aaron committed:
Corrección del programa

Utilice el método replace para eliminar donde habían dos espacios para poder solucionar el problema con los espaciados
